src/controllers/post.controller.js

```diff
@@ -24,58 +25,65 @@ const addNewPost = asyncHandler(async (req, res, next) => {
   res.status(201).json(new ApiResponse(201, { newPost }, "Post created"));
 });
 
-// ------------------------- Update Post -------------------------
+// ------------------------- Update self Post -------------------------
 const updatePost = asyncHandler(async (req, res, next) => {
-  const { postId, image, caption, likes, comments } = req.body;
+  const { postId, caption } = req.body;
```

Comment:
What if user does not want to update caption but just image ?

Reply:
user cannot update image. i dont want to add such feature